setupext.py

```diff
@@ -627,6 +627,8 @@ def do_custom_build(self, env):
                 },
                 **env,
             }
+            if os.path.exists(os.path.join(src_path, "autogen.sh")):
```

Comment:
I think this logic should be "If the results of autogen.sh is not there, run autotools"

Reply:
I don't think that logic is needed for  that i think  if autogen.sh is not there, which means that it is not necessary for the project.